moveit_core/collision_detection_bullet/include/moveit/collision_detection_bullet/bullet_integration/bullet_utils.h

```diff
@@ -58,42 +58,8 @@ const bool BULLET_COMPOUND_USE_DYNAMIC_AABB = true;
 MOVEIT_CLASS_FORWARD(CollisionObjectWrapper)
 
 /** \brief Allowed = true */
-inline bool acmCheck(const std::string& body_1, const std::string& body_2,
-                     const collision_detection::AllowedCollisionMatrix* acm)
-{
-  collision_detection::AllowedCollision::Type allowed_type;
-
-  if (acm != nullptr)
-  {
-    if (acm->getEntry(body_1, body_2, allowed_type))
-    {
-      if (allowed_type == collision_detection::AllowedCollision::Type::NEVER)
-      {
-        ROS_DEBUG_STREAM_NAMED("collision_detection.bullet", "Not allowed entry in ACM found, collision check between "
-                                                                 << body_1 << " and " << body_2);
-        return false;
-      }
-      else
-      {
-        ROS_DEBUG_STREAM_NAMED("collision_detection.bullet", "Entry in ACM found, skipping collision check as allowed "
-                                                                 << body_1 << " and " << body_2);
-        return true;
-      }
-    }
-    else
-    {
-      ROS_DEBUG_STREAM_NAMED("collision_detection.bullet",
-                             "No entry in ACM found, collision check between " << body_1 << " and " << body_2);
-      return false;
-    }
-  }
-  else
-  {
-    ROS_DEBUG_STREAM_NAMED("collision_detection.bullet",
-                           "No ACM, collision check between " << body_1 << " and " << body_2);
-    return false;
-  }
-}
+bool acmCheck(const std::string& body_1, const std::string& body_2,
```

Comment:
I agree that this should be moved to the cpp files. @j-petit could you comment on any potential drawbacks with this change?

Reply:
I think this is fine to move. I simply used the structure which was given in the old tesseract / bullet code.